crates/store/src/db/migrations.rs

```diff
@@ -75,6 +75,20 @@ pub fn apply_migrations(conn: &mut Connection) -> super::Result<()> {
         Settings::set_value(conn, DB_MIGRATION_HASH_FIELD, &new_hash)?;
     }
 
+    info!(target: COMPONENT, "Starting database optimization");
+
+    // Run full database optimization. This will shrink the database and run indexes analysis for
+    // the query planner. This will also increase the `schema_version` value.
```

Comment:
This does _not_ shrink the database, it just marks them for re-analysis for optimisation.

```
0x00002 | Run ANALYZE on tables that might benefit. On by default.
          
0x10000 | Check the size of all tables, not just tables that have not been recently
        | used, to see if any have grown and shrunk significantly and hence might 
        | benefit from being re-analyzed. Off by default.
```